internal/cli/internal/command/volumes/delete.go

```diff
@@ -0,0 +1,61 @@
+package volumes
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/cli/internal/prompt"
+	"github.com/superfly/flyctl/internal/client"
+	"github.com/superfly/flyctl/pkg/iostreams"
+)
+
+func newDelete() *cobra.Command {
+	const (
+		long = `Delete a volume from the application. Requires the volume's ID
```

Comment:
This reads "delete a volume from the application" but it takes a volume id. Perhaps these dox should be changed to better reflect what the command does.